src/python/src/grpc/_adapter/_c/__init__.pyx

```diff
@@ -0,0 +1,992 @@
+# vim: expandtab:ts=2:sw=2:sts=2
+
+cimport grpc
+cimport cpython
+
+import time
+
+# There are a lot of nested structures in GRPC C core. We wrap all of them in
+# Cython extension classes and nest them in parallel in their C and Python
+# representations. Because they memory is synchronized between them, we can
+# forego a lot of the manual memory management that would be handled by using
+# the CPython API alone. Furthermore, we don't need to handle as many GC issues
+# (any GC issues at all?) because of this interlocking between C and Python
+# memory.
+
+#
+# Global state
```

Comment:
That was actually a note to myself, but I was still intending on having a section header. There's no Cython-specific reason, but there is a memory-management-awareness reason. There's at least one instance of cross-cutting (via `OperationTag` between `CompletionQueue`, `Server`, `Call`), and even if the file were broken up I'd still prefer to have those classes together (because memory management is pretty much the only important thing in the wrapping layer). They weren't together before in CPython because the file length would've been long enough that one could not easily skim through it in text editors without block folding.